controller/service.go

```diff
@@ -260,3 +260,12 @@ func isEqualIPs(ipsA, ipsB []net.IP) bool {
 	})
 	return reflect.DeepEqual(ipsA, ipsB)
 }
+
```

Comment:
shouldn't we have this function in ipfamily package ?

Reply:
The reason why I put this here was that the ipfamily package is (was) about ips and families and independent from k8s concepts, so it's a bit borderline and felt more like a local util function.
I don't have a strong opinion though, `ipfamily.ForService` sounds as much as good. Will move it.

Reply:
heh, I started a similar comment, then realized the ipFamily pkg was agnostic of k8s and decided it was fine too.

I am all for `ipfamily.ForService`, but if you do it and decide it doesn't fit, I'm ok with how the code currently is too :)

Reply:
I just pushed the change :-)